modules/ROOT/pages/_partials/commons/common-vector-search.adoc

```diff
@@ -0,0 +1,217 @@
+The search results are then included as context data to queries sent to the LLM in order to customize query responses. 
+This results in the LLM returning a result that is likely to be far more contextually relevant than the standalone prompt.
+
+Additionally, Vector Search in Couchbase Lite provides the following benefits:
+
+* Unified Cloud-to-Edge Support for Vector Similarity Search - Couchbase supports Vector Search from cloud to edge, which enables applications to efficiently utilize cloud and edge computing's strengths.
+
+* Enhanced Data Privacy on the Edge - By performing Vector Search within the device, personal data and search queries of a sensitive nature do not have to leave the device.
+
+* Low Latency Application Support - You can run searches locally against a local dataset using a local embedded model. This eliminates the network variability and results in more consistent execution speed.
+Even in the case where the model is not embedded within the local device but is deployed at the edge location, the round trip time (RTT) associated with queries can be significantly reduced compared to searches made over the Internet.
+
+* Cost Per Query Reduction - When you have hundreds of thousands of connected clients querying against a cloud-based LLM, the load on cloud model and operational costs of running the cloud based model can be considerably high.
+By running queries locally on the device, you can save on data transfer costs and cloud egress charges while also decentralizing the operational costs.
+
+== Key Concepts of Vector Search in Couchbase Lite 
+
+When xref:{param-module}:working-with-vector-search.adoc[working with Vector Search], you should be aware of the core concepts below. 
+
+=== About Vector Embeddings
+
+Vector embeddings represent the output of a Machine Learning (ML) model as an array of numbers to capture semantic or contextual relationships between data points.
+This representation encodes how a ML model understands the input or inputs provided to it, based on how the model was initially trained and the internal structure of the model.
+When a model considers the features of a given input as similar, the distance between the vector embeddings will be short.
+Vector embeddings are stored within embedded vector indexes.
+
+The current supported formats for Vector embeddings are:
+
+* An array of 32-bit floats.
+
+* A Base64 string that encodes a https://en.wikipedia.org/wiki/Endianness[Little-endian] array of 32-bit floats.
+
+=== About Vector Indexes
+
+Vector Indexes are used to store and manage vector representations of content in the form of vector embeddings.
+You can use Vector Indexes to efficiently retrieve vectors, similar to a target vector.
+Before use, a Vector Index needs to be trained to compute the centroids and parameters for encoding the vectors.
```

Comment:
So, you mention the "minimum-training-size' configuration in the paragraph below. Is this something that the customer sets? Where is it and how do they set it?

If it is indeed under the control of the user, I'd suggest changing to something like:

Before use, a Vector Index needs to be trained to compute the centroids and parameters for encoding the vectors. You configure the number of vectors required in the index for training to take place using the minimum training size option[[[specify where and how]]. 

If the database contains enough vectors, Index training begins automatically when the first Vector Search query is performed on the database. (If the database does not contain the required number of vectors, an error message appears in the log.)

Reply:
Done! I've opted to point to the working with doc for context-relevant snippets displaying how you can configure training sizes and to point to the configuration table so users can view possible configuration options.